types/luxon/src/weekSettings.d.ts

```diff
@@ -0,0 +1,7 @@
+import { WeekdayNumbers } from "./datetime";
+
+export interface WeekSettings {
```

Comment:
I would just keep this in the settings file. It's not really applicable anywhere else currently and the js src doesn't have this file.

Reply:
Landed in [7828fd4](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68088/commits/7828fd4198e62d415792b7abdc7218154e5074fd)